test/functional/rpc_setban.py

```diff
@@ -22,7 +22,7 @@ def run_test(self):
         # Node 0 connects to Node 1, check that the noban permission is not granted
         self.connect_nodes(0, 1)
         peerinfo = self.nodes[1].getpeerinfo()[0]
-        assert(not 'noban' in peerinfo['permissions'])
+        assert not "noban" in peerinfo["permissions"]
```

Comment:
fe4aafda512483e863b23cfad403509b1604555c drive-by nit, while touching lines 25 and 46 in this commit

`$ pycodestyle test/functional/rpc_setban.py`

    test/functional/rpc_setban.py:25:16: E713 test for membership should be 'not in'
    test/functional/rpc_setban.py:46:16: E713 test for membership should be 'not in'

Reply:
Going to leave it as is because none of "black" or "pycodestyle" is mentioned in `doc/`. Don't want to bother other reviewers with too big and unrelated changes to this PR.

Reply:
Ok. For info, pycodestyle is PEP8 that we follow per doc/test/functional/README.md

https://pypi.org/project/pycodestyle

Reply:
(ah, it was renamed from PEP8 for this reason https://github.com/PyCQA/pycodestyle/issues/466)

Reply:
`lint-python.sh` is happy with the current `rpc_setban.py`:

```sh
$ ./test/lint/lint-python.sh
test/functional/test_runner.py:42: error: Module has no attribute "getwindowsversion"
Found 1 error in 1 file (checked 199 source files)
$
```

Reply:
If adding new code or retouching old code for style reasons, I'd try to make it conform (https://www.flake8rules.com/rules/E713.html for "not in") because reviewers are often picky about it. I also wouldn't add a commit that does a one-word grammar change because otherwise someone will invariably ask me to squash the commit. But everyone has a different experience with this so definitely feel free to ignore.